src/Assert.php

```diff
@@ -813,6 +832,42 @@ public static function count($array, $number, $message = '')
         );
     }
 
+    public static function minCount($array, $min, $message = '')
+    {
+        if (count($array) < $min) {
```

Comment:
Not sure I have to apply `isCountable` assertion here. Look at the `count` helper: there is no assertion about countability there. Same thing in `length`, `minLength`, `maxLength` assertion: there is no assertion about type of the argument there. So I think it's better not to use `isCountable` assertion to keep the behavior consistent across the library